swagger.yaml

```diff
@@ -3014,21 +3110,29 @@ components:
           $ref: '#/components/schemas/BillingConfigurationCustomer'
     Customer:
       type: object
+      required:
+        - customer
       properties:
         customer:
           $ref: '#/components/schemas/CustomerObject'
     Customers:
       type: object
+      required:
+        - customers
       properties:
         customers:
           type: array
           items:
             $ref: '#/components/schemas/CustomerObject'
     CustomerInput:
       type: object
+      required:
+        - customer
       properties:
         customer:
           type: object
+          required:
+            - external_id
```

Comment:
The same endpoint is used for both create and update so external_id should be optional parameter